docs/examples/kelvin_helmholtz.jl

```diff
@@ -163,6 +163,6 @@ end
 # ![](kelvin_helmholtz.mp4)
 #
 # Similarly to the kinetic energy dissipation rate (see the [Two-dimensional turbulence example](@ref two_d_turbulence_example)),
-# `TracerVarianceDissipationRate` and `TracerVarianceDiffusiveTerm` are implemented
+# `TracerVarianceDissipationRate` and `TracerVarianceDiffusion` are implemented
 # with a energy-conserving formulation, which means that (for `NoFlux` boundary conditions) their
```

Comment:
with an energy-conserving formulation, which means that for `NoFlux` boundary conditions their

Reply:
Sorry @jbisits not sure I follow your comment